keyboard:
  comp_keyboard:
    known_key: 'z'
    character_deltas:
      a:
        x_delta: -1.0
        y_delta: 0.0
        z_delta: 1.5
      b:
        x_delta: 7.5
        y_delta: 0.0
        z_delta: 0.0
      c:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 4.0
      d:
        x_delta: 3.0
        y_delta: 0.0
        z_delta: 2.0
      e:
        x_delta: 2.5
        y_delta: 0.0
        z_delta: 4.0
      f:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      g:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      h:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      i:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      j:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      k:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      l:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      m:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      n:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      o:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      p:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      q:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      r:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      s:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      t:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      u:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      v:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      w:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      x:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      y:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
      z:
        x_delta: 0.0
        y_delta: 0.0
        z_delta: 0.0
